.github/workflows/c-cpp.yml

```diff
@@ -0,0 +1,76 @@
+name: C/C++ CI
```

Comment:
Consider adding Bionic / Linux to the name

Reply:
fb13a87836cc668df9275cd7b17e3baf3d289c07